core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1027,12 +1027,21 @@ private[spark] object Utils extends Logging {
   }
 
   def checkHost(host: String): Unit = {
```

Comment:
This function semantic is changed to handle `IP`. Could you add a function description to explain it?